addons/binding/org.openhab.binding.amazonechocontrol/META-INF/MANIFEST.MF

```diff
@@ -1,45 +0,0 @@
-Manifest-Version: 1.0
-Automatic-Module-Name: org.openhab.binding.amazonechocontrol
-Bundle-ActivationPolicy: lazy
-Bundle-ClassPath: .
-Bundle-ManifestVersion: 2
-Bundle-Name: Amazon Echo Control Binding
-Bundle-SymbolicName: org.openhab.binding.amazonechocontrol;singleton:=true
-Bundle-RequiredExecutionEnvironment: JavaSE-1.8
-Bundle-Vendor: openHAB
-Bundle-Version: 2.5.0.qualifier
-Import-Package: com.google.gson,
- javax.measure,
- javax.measure.quantity,
- javax.servlet,
- javax.servlet.http,
- javax.ws.rs.core,

```

Comment:
@mgeramb Does the binding really depend on `javax.ws.rs.core`? Just asking because it maybe is important for #5162 .

Reply:
I didn’t see any import from that at first glance.